docs/gadget-devel/metrics.md

```diff
@@ -17,7 +17,7 @@ Metrics can either be collected in user-space or directly inside your eBPF progr
 choose depends on how you collect the data and the quantity of it.
 
 If you're sending events to user-space, you can create metrics from those by just adding a couple of annotations to your
-`gadget.yaml` file, or by using some well-known types (TODO links to macros) inside your struct definition inside eBPF code.
+`gadget.yaml` file, or by using some well-known types inside your struct definition inside eBPF code.
 
 If you don't want to emit events, because it would just be too much throughput, you can choose to write the metrics into
 eBPF maps instead and let IG create a data source (TODO link to data source) from it. Those can then be
```

Comment:
nit: Add link to corresponding sections: Using the metadata file, using well-known types and "Collection of metrics from eBPF maps".

Reply:
Where should add these links?